src/com/autoembed/youtube.less

```diff
@@ -1,101 +1,39 @@
 @import "defs.less";
+@import '../../custom/brandcolors/brandcolors.less';
 
-#content .embed-video {
-  max-width: 640px;
-  margin: auto;
-  position: relative;
-  cursor: pointer;
+/*
+* Overrides 'video-embeds' styling with youtube specific brand colors.
+*/
 
-  & > .-thumbnail {
-    position: relative;
+#content .embed-video.-youtube {
 
-    & > .-overlay {
-      position: absolute;
-      width: 100%;
-      height: 100%;
-      top: 0;
-      left: 0;
-      z-index: 10;
+    & > .-thumbnail {
 
-      & > .-play {
-        position: absolute;
-        top: 50%;
-        left: 50%;
-        transform: translate(-50%,-50%);
-        font-size: 3em;
-        color: @COL_L;
-        background: fade(@COL_D, 50%);
-        transition: all 0.2s;
+        & > .-overlay {
 
-        & > svg {
-          display: block;
-        }
-      }
-      /* while hovering */
-      &:hover > .-play {
-        font-size: 4em;
-        background: @COL_A;
-      }
-      /* squish effect on click */
-      &:active > .-play {
-        font-size: 2.5em;
-      }
-
-      & > .-external {
-        position: absolute;
-        bottom: 1rem;
-        right: 1rem;
-        font-size: 1.5em;
-        color: @COL_L;
-        opacity: 0;
-        transform: scale(1, 1);
-        transition: all 0.2s;
-        padding: 0.5rem;
-        background: fade(@COL_D, 50%);
-
-        &:hover {
-          transform: scale(1.5, 1.5);
-          background: @COL_A;
-        }
-
-        & > a {
-          padding: 0;
-          line-height: 1;
-        }
+            &:hover > .-play {
+                color: @COL_L;
+                background: @bc-youtube;
+            }
 
-        & > a,
-        & > a:visited {
-          color: @COL_L;
-        }
-      }
+            &:active > .-play {
 
-      &:hover > .-external {
-        opacity: 1;
-        /*color: @COL_D;*/
-      }
-    }
+            }
 
-    & > img {
-      width: 100%;
-      height: auto;
-      display: block;
-    }
+            & > .-external {
 
-    &:hover {}
-  }
+                color: @bc-youtube;
+                background: fade(@COL_D, 50%);
 
-  & > .-video {
-    position: relative;
-    width: 100%;
-    height: 0;
-    padding-bottom: 100% * (9/16);
+                &:hover {
+                background: @bc-youtube;
+                }
 
-    & > iframe {
-      position: absolute;
-      width: 100%;
-      height: 100%;
-      left: 0;
-      top: 0;
+                & > a,
+                & > a:visited {
+                color: @COL_L;
+                }
+            }
+        }
     }
-  }
-}
+}
```

Comment:
This should have been caught by the linter. Newline at end of file.

Reply:
fixed